src/main/java/net/minecraftforge/client/event/TextureStitchEvent.java

```diff
@@ -38,17 +41,29 @@ public AtlasTexture getMap()
     }
 
     /**
-     * Fired when the TextureMap is told to refresh it's stitched texture. 
-     * Called after the Stitched list is cleared, but before any blocks or items
-     * add themselves to the list.
+     * Fired when the TextureMap is told to refresh it's stitched texture.
+     * Called before the {@link net.minecraft.client.renderer.texture.TextureAtlasSprite} are loaded.
+     * Contains the set of {@link ResourceLocation}s that will load TextureAtlasSprites.
+     *
+     * To load new textures to be stitched into the AtlasTexture add to the resourceLocations Set.
```

Comment:
This sentence should be removed and replaced with a javadoc for the `addResourceLocation()` method below.